shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/map/StopLayerGenerator.kt

```diff
@@ -38,14 +39,22 @@ object StopLayerGenerator {
                 Exp.image(Exp(""))
             )
         stopSelectedPinLayer.iconOffset = offsetPinValue()
-        includeSharedProps(stopSelectedPinLayer)
+        stopTouchTargetLayer.filter =
+            Exp.ge(
+                Exp.zoom(),
+                Exp.case(
+                    MapExp.isBusExp to Exp(busStopZoomThreshold + 1),
+                    Exp(stopZoomThreshold + 1)
```

Comment:
These +1s feel counterintuitive to me - can you add a comment explaining why they're necessary?

Reply:
Double checked my logic and I think we don't need the + 1's at all - minZoom will hide the layer if zoom < threshold, and filter will show the layer if zoom >= threshold, so the threshold shouldn't need the adjustment